deploy/install.sh

```diff
@@ -7,9 +8,42 @@ else
     echo 'Docker is already installed.'
 fi
 
-docker compose up --build frontend_build
-docker compose up --build documentation_build
-docker compose up -d --build db
-docker compose up -d --build backend
-docker compose up -d --build worker
-docker compose up -d --build frontend
+
+
+# Function to start SSL registration with Certbot and Letsencrypt SSL
+ssl_registration () {
+    source .env
+    # Obtain domain name from .env file
+    domain=$(echo $WEBDOMAIN | awk -F/ '{print $3}')
```

Comment:
Modify documentation https://github.com/akvo/wai-sdg-portal/blob/main/docs/wai-demo/source/install.rst#environment-variable-setup. Include `http` even if it's a localhost domain